src/components/form/ValidatedFranceconnectEnrollmentsSelector.js

```diff
@@ -141,11 +132,17 @@ class ValidatedFranceconnectEnrollmentsSelector extends React.Component {
         {noValidatedFranceconnectEnrollments && (
           <div className="form__group">
             <div className="notification error">
-              Veuillez{' '}
-              <Link to={'/franceconnect'}>
-                demander votre accès à Franceconnect
-              </Link>{' '}
-              avant de continuer cette demande.
+              <p>
+                Pour demander l'accès à l'API « impôt particulier », vous devez
```

Comment:
I am not sure that the FS will understand why he should specify a validated franceconnect enrollement

Reply:
still there